### ISSUE TITLE
[a11y] The error pages ("Address Not Found") aren't read

### ISSUE THREAD

**Issue by lobontiumira** [mozilla-mobile/focus-android#6135]
## Steps to reproduce
1. Enable talkback or screen reader on your device.
2. Open Focus, and load an incorrect URL, like ttt.com.
3. Listen the screen reader.

### Expected behavior
The screen reader announces the "Address not found" page.

### Actual behavior
The screen reader does not announce a thing. After the page is loaded, nothing is read.

### Device information
* Android device: Google Pixel (Android 10)
* Focus version: Focus Beta 96.0.0-beta.5

**Comment by iorgamgabriel** [comment]
I couldn't reproduce this bug . Tried on Pixel 3XL android 12 , OnePlus 7t android 11 , Motorola Moto E4 android 7

**Comment by lobontiumira** [comment]
I tested again on the latest Focus Nightly 101.0a1 build from 4/27, and the Talkback announces only the "Erase browsing history button. Double-tap to activate." on Oppo Reno 6 (Android 12).
On the Samsung Galaxy Note 8 (Android 9), the voice assistant doesn't announce anything.

**Comment by lobontiumira** [comment]
I've tested with some more devices, on the 12.2 version of Talkback, and the behavior is different for each device:
- on Oppo Reno 6 (Android 12), the "Address not found" is announced on Focus, and on Fenix,
- on Lenovo tablet (Android 10), the "Address not found" is announced only on Fenix, and not on Focus,
- on Xiaomi Mi8 Lite (Android 9), the "Address not found" isn't announced at all, on neither Focus or Fenix.
Tested on the latest Focus Nightly, and Fenix Nightly builds.

**Comment by ionutbedregeanu** [comment]
Put this back to backlog since are a few more details to discuss with the QA team

**Comment by lobontiumira** [comment]
Hello,
I've retested with today's builds of Focus Nightly 102.0a1, Fenix 102.0a1, and Chrome with the following devices:
* Oppo Reno 6 (Android 12), the "Address not found" is not announced anymore neither on Focus, nor on Fenix (with the tollbar set to the top),
* Sony Xperia Z5 Premium (Android 7.1.1), the "Address not found" is not announced neither on Focus, nor on Fenix (with the tollbar set to the top),
* Xiaomi Mi8 Lite (Android 10), the "Address not found" is not announced neither on Focus, nor on Fenix (with the tollbar set to the top).

I'll close this ticket.
